backend/app/tests/ui/test_e2e_web_preview_git.py

```diff
@@ -33,6 +36,19 @@ def test_e2e_web_preview_git(chrome_page, corgi_base_url, colid, version, style)
     home.remove_focus()
     home.click_create_button()
 
-    # THEN: The home closes and job is queued
-    assert home.create_new_job_button_is_visible
-    assert "queued" in home.status_message.text_content()
+    # AND: Data from latest job are collected
+    response_json = requests.get(url).json()
```

Comment:
Your going ahead and decoding to json but you may want check and make sure you receive a 200 before continuing with your test.

Reply:
ok. I'll add this